### PR TITLE
Improve fall motion

### PR DESCRIPTION
**Pull request by Vivituhh**
## Introduced Changes

When a NAO is falling, it currently doesn't do much except reducing the stiffness and moving the head slightly.

The new falling motion contains bending the knees and unrolling. This should reduce the strain on impact.

## Ideas for Next Iterations (Not This PR)

Create the motion such that it brings the NAO into a position more favourable for the following stand-up motion.

## How to Test

Test in Webots: Start Webots with `webots/worlds/falling.wbt` and start the simulation. Robot should then do the motion in the next 5s.

Test in Lab with Robot: Push the robot. Moreover it was tested in a test game.

### CODE REVIEW

File: crates/control/Cargo.toml [hunk 1]
```diff
@@ -7,6 +7,7 @@ homepage = "https://github.com/hulks/hulk"
 
 [dependencies]
 approx = { workspace = true }
+byteorder = { workspace = true }
```
Comment (MaikRe) [inline crates/control/Cargo.toml:10]: Make sure to rebase, this dependency was removed from the webots falling branch before it was merged as it is unused

File: etc/configuration/default.json [hunk 2]
```diff
@@ -633,7 +633,7 @@
     "angular_velocity_low_pass_factor": 0.2,
     "roll_pitch_low_pass_factor": 0.2,
     "gravitational_acceleration_threshold": 4.0,
-    "falling_angle_threshold": [0.5, 0.45],
+    "falling_angle_threshold": [0.5, 0.4],
```
Comment (MaikRe) [inline etc/configuration/default.json:636]: These are changed by #212 which is already approved for merge, you may need to adjust accordingly and potentially retest to make sure functionality remains as expected

File: crates/control/Cargo.toml (hunk 1 above)
Comment (Vivituhh) [reply crates/control/Cargo.toml]: Done

File: etc/configuration/default.json (hunk 2 above)
Comment (Vivituhh) [reply etc/configuration/default.json]: Done. It is changed and the motion is adapted to this parameter.

**Review by julianschuler — APPROVED** [review]

**Pull request merged**